io_scene_niftools/modules/nif_export/geometry/mesh/__init__.py

```diff
@@ -265,6 +279,9 @@ def export_tri_shapes(self, b_obj, n_parent, trishape_name=None):
                         vertlist.append(vertquad[0])
                         if mesh_hasnormals:
                             normlist.append(vertquad[2])
+                        if use_tangents:
+                            tanlist.append(b_mesh.loops[loop_index].tangent)
```

Comment:
also, if we use tangents we need to have normals so we should perhaps nest the if?

Reply:
Tanlist and bitlist names were taken from the other lists that already existed (vertlist, normlist, vcollist, uvlist, trilist). If you want to change that naming scheme, it can certainly be done, but I think that might be better packaged up in a future PR.

Reply:
I thought since you're adding new lists you might as well fix the naming scheme there. ;)

Reply:
Either or, I know my dictionary checker will like having the underscores or better names.
Don't mind if done as part of this or subsequent PR.